src/components/shared/SilderMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { Layout as LayoutBase, Menu } from 'antd';
+import { useNavigate } from 'react-router-dom';
+import { HomeOutlined, BookOutlined } from '@ant-design/icons';
+import { ROUTES } from '../../constants/common';
+import type { MenuProps } from 'antd';
+
+const { Sider } = LayoutBase;
+
+const SilderMenu = () => {
+  const navigate = useNavigate();
+  const [currentLocation, setCurrentLocation] = useState(window.location.pathname);
+
+  const menuItems = [
+    { key: ROUTES.HOME, icon: <HomeOutlined />, label: 'Home' },
+    { key: ROUTES.BOOKINGS, icon: <BookOutlined />, label: 'Bookings' },
+  ];
+
+  const onClick: MenuProps['onClick'] = (e) => {
```

Comment:
It's common approach to use `e` for events but let's avoid short aliases for function parameters. `event.key` for example is much more readable than `e.key`. So there's nothing wrong here but I suggest to use full names